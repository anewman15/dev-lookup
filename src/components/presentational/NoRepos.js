import PropTypes from 'prop-types';

const NoRepos = ({ username }) => (
  <div>
    <p>
      No repositories available for&nbsp;
      <strong>
        {username}
      </strong>
    </p>
  </div>
);

NoRepos.propTypes = {
  username: PropTypes.string,
}.isRequired;

export default NoRepos;
