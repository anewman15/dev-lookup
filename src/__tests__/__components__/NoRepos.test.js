import { unmountComponentAtNode } from 'react-dom';
import renderer from 'react-test-renderer';
import NoRepos from '../../components/presentational/NoRepos';

let container = null;
beforeEach(() => {
  container = document.createElement('div');
  document.body.appendChild(container);
});

afterEach(() => {
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

describe('NoRepos', () => {
  it('renders the NoRepos component', () => {
    const tree = renderer
      .create(<NoRepos />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
