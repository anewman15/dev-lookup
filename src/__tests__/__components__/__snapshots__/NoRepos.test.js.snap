// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoRepos renders the NoRepos component 1`] = `
<div
  className="container flexCenter"
>
  <p
    className="boxBlack"
  >
    No repositories available forÂ 
    <strong />
  </p>
</div>
`;
